lib/index.ts

```diff
@@ -165,6 +165,7 @@ export module python {
   }
 }
 
+export { astHelpers } from "../python/index.js";
```

Comment:
I would prefer this not be called `astHelpers`, because it does not suggest it is just for Python.
What about including it in the `python` module?

Reply:
Sounds good. I'll update that when I get webpack to build this.

If it's in the python module, is `astHelpers` okay as a name?

Reply:
> If it's in the python module, is astHelpers okay as a name?

Yes, that hierarchy makes sense to me.